holoviews/plotting/bokeh/widgets.py

```diff
@@ -95,52 +93,80 @@ def create_widget(self, dim, holomap=None, editable=False):
         label, mapping = None, None
         if holomap is None:
             if dim.values:
+                if dim.default is None:
+                    default = dim.values[0]
+                elif dim.default not in dim.values:
```

Comment:
Isn't this already validated at the dimension level?

Reply:
Yes, at least the dynamic cases are validated now.